static/stable/web_help/docs/zowe_zos-jobs_cancel_job.html

```diff
@@ -22,6 +22,19 @@ <h4>Positional Arguments</h4>
 </ul>
 </li>
 </ul>
+<h4>Options</h4>
+<ul>
+<li>
+<p><code>--modifyVersion</code>  <em>(string)</em></p>
+<ul>
+<li>
+<p>Using this option to set X-IBM-Job-Modify-Version to &quot;2.0&quot; will make the
```

Comment:
This is kind of hard to understand, so let me try to unpack the meaning. Does one only use the `--modifyVersion` option to set the `X-IBM-Job-Modify-Version` parameter to `2.0` (value)? And by doing so, the cancel job API becomes synchronous? The current description sounds as if this is just one use case. Can this option be used for something else?

Reply:
I wanted to suggest some change but I cannot do that based on my current understanding of what this option does